src/components/Note/NoteReactionList/NoteReactionBar/NoteReactionBar.test.tsx

```diff
@@ -29,8 +29,10 @@ describe("NoteReactionBar", () => {
   it("focuses the first emoji button after TAB is pressed on the last emoji button", async () => {
     render(<NoteReactionBar closeReactionBar={() => {}} reactions={[]} handleClickReaction={() => {}} />);
 
-    const firstEmojiButton = screen.getByText([...REACTION_EMOJI_MAP.values()].at(0)!);
-    const lastEmojiButton = screen.getByText([...REACTION_EMOJI_MAP.values()].at(-1)!);
+    // @ts-ignore
+    const firstEmojiButton = screen.getByText([...REACTION_EMOJI_MAP.values()].at(0).emoji!);
```

Comment:
add a `!` after `.at(0)` to assert the value is defined, then you can remove the ts-ignore

Reply:
> Why is the skintone just stored locally btw? Would make more sense for it to be another db column in users imo, and then be retrieved from store.user. That way, other people can also see the respective skin colors, which makes more sense if you ask me.

I thought the same about this.
Would definitely implement this

Reply:
the problem is when i remove the ts-ignore it gets displayed as an error and the tests still fail

Reply:
that's a different issue though, seems like a missing `Provider` is the problem, you can see how it's solved in other tests